Assets/Scripts/Game/LoanChecker.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+using System.Collections.Generic;
+using DaggerfallConnect.Arena2;
+using DaggerfallWorkshop.Game.Banking;
+using DaggerfallWorkshop.Game.Entity;
+using DaggerfallWorkshop.Utility;
+using UnityEngine;
+
+namespace DaggerfallWorkshop.Game
+{
+    public class LoanChecker : MonoBehaviour
```

Comment:
Does this class need to be a MonoBehaviour? It doesn't appear to be added to any GameObjects, and you're just calling the static methods inside of it. Could drop the MonoBehaviour and perhaps make the whole class static.

Reply:
Done